app/src/components/main/grid/gridBody/gridRow/gridRow.jsx

```diff
@@ -54,6 +74,37 @@ export class GridRow extends Component {
     this.setState({ withAccordion: true });
     this.overflowCell.style.maxHeight = `${this.overflowCellMaxHeight}px`;
   };
+
+  getHighLightBlockHeight = () => {
+    const { rowClassMapper, value } = this.props;
+    if (
+      this.state.withAccordion &&
+      !this.state.expanded &&
+      rowClassMapper &&
+      !rowClassMapper(value)['row-console']
```

Comment:
`rowClassMapper` is not meant to use like that.

Reply:
Fixed